src/components/deviceprofileBuilder.ts

```diff
@@ -302,6 +329,31 @@ function getTranscodingProfiles(): Array<TranscodingProfile> {
     }
 
     const hlsVideoCodecs = getSupportedHLSVideoCodecs();
+
+    if (hlsVideoCodecs.length && surroundCodecs.length) {
+        // Direct streaming for passthrough codecs
```

Comment:
I haven't commented clearly enough that chromecast does not really support this and it may fail on some files.